module_utils/network_lsr/argument_validator.py

```diff
@@ -2216,6 +2216,17 @@ def _ipv6_is_not_configured(connection):
                     idx,
                     "ip.ipv6_disabled is not supported by initscripts.",
                 )
+            if not hasattr(Util.NM(), "SETTING_IP6_CONFIG_METHOD_DISABLED"):
```

Comment:
if you run on a system with initscripts backend, then you cannot assume to have `Util.NM()`. Check with
`if mode == self.VALIDATE_ONE_MODE_NM:`.

Reply:
If you looked at the code holistically, the `Util.NM()`  will only be called when the mode is `self.VALIDATE_ONE_MODE_NM`, therefore, I think that the additional check is not needed. 
```python
        # initscripts does not support ip.ipv6_disabled, so raise errors when network
        # provider is initscripts
        if connection["ip"]["ipv6_disabled"]:
            if mode == self.VALIDATE_ONE_MODE_INITSCRIPTS:
                raise ValidationError.from_connection(
                    idx,
                    "ip.ipv6_disabled is not supported by initscripts.",
                )
            if not hasattr(Util.NM(), "SETTING_IP6_CONFIG_METHOD_DISABLED"):
                raise ValidationError.from_connection(
                    idx,
                    "ip.ipv6_disabled is not supported by the running version of "
                    "NetworkManager, it requires at least version 1.20. But you can "
                    "disable IPv6 auto configuration by setting ip.auto6 to False. "
                    "Then NetworkManager will ignore IPv6 for this connection. This "
                    "will still leave the sysctl value 'disable_ipv6' unchanged, but "
                    "setting ip.ipv6_disabled to True in the role will set the sysctl "
                    "value 'disable_ipv6' to True ",
                )
```

Reply:
you are right. I didn't see!!